From 6d2edb1635465dc226bc7e05fd33c9ee456e2430 Mon Sep 17 00:00:00 2001
From: Jamey Sharp <jamey@minilop.net>
Date: Mon, 30 Jul 2018 13:26:28 -0700
Subject: [PATCH 1/3] Make libide's install_dir an absolute path.

As far as I know, this change should be a no-op on all systems where
Builder works today. However, it is necessary for getting the correct
shared-library path into generated .gir and typelib files when building
on Nix, so I'm hoping this is an acceptable change for upstream.
---
 src/libide/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libide/meson.build b/src/libide/meson.build
index e0643eda4..e27e0e43a 100644
--- a/src/libide/meson.build
+++ b/src/libide/meson.build
@@ -229,7 +229,7 @@ libide = shared_library('ide-' + libide_api_version,
   dependencies: libide_deps,
         c_args: libide_args,
        install: true,
-   install_dir: pkglibdir,
+   install_dir: pkglibdir_abs,
  install_rpath: pkglibdir_abs,
 )
 
-- 
2.16.4

